### DISCUSSION TITLE
Authors of package?

### DISCUSSION THREAD

**Discussion by jlewis97** [heli-xu/findSVI#14] [General]
Hello, I am doing a project that utilizes this package. I am curious if the authors are affiliated with CDC/ATSDR. Also, are there any references validating the methods for these functions? Thanks!

**Comment by heli-xu** [comment]
Hi, thank you for your interest in the package. We are researchers at the Urban Health Collaborative at Drexel University and not affiliated with CDC/ATSDR. The methods for these functions are based on [CDC/ATSDR SVI documentation](https://www.atsdr.cdc.gov/placeandhealth/svi/data_documentation_download.html), which are also described in these two papers from CDC/ATSDR ([Flanagan et al. 2011](https://svi.cdc.gov/A%20Social%20Vulnerability%20Index%20for%20Disaster%20Management.pdf) and [Flanagan et al. 2018](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7179070/)). There is a [vignette](https://heli-xu.github.io/findSVI/articles/svi-validation.html) about validating the SVI results from these functions by comparing them with the SVI released by CDC/ATSDR, and we are working on a paper on the package. 
Hope this helps, and feel free to let us know if you have other questions/feedback. Thanks!

**Reply by jlewis97** [reply]
Thank you for the response! Might I recommend that it may be beneficial to have variable calculations for the `EP_ SVI variables` to only use the corresponding "E" census variables instead of the "EP" census variables. For example, `EP_MOBILE` is calculated as `DP04_0014PE`, but it could instead be calculated as `DP04_0014E/ (DP04_0001E) * 100`. That way, users could further aggregate to geographic levels that are not supported by the U.S. Census (e.g., ZCTA3). For example, a potential workflow could be `get_census_data(geography = "zcta"`, followed by grouping/summing all the census variables by the first three digits of the zcta, and then using `get_SVI()` to calculate SVI at the new geography. This workflow, however, is not supported with the current calculations used in `variable_e_ep_calculation_` because of the use of `EP census variables`. 

Regardless, thank you for your work on this package! It has been very helpful in my research!

**Reply by heli-xu** [reply]
Thank you for your suggestion! That's a good point. The current version of `variable_e_ep_calculation` tables are directly extracted from CDC/ATSDR SVI dictionaries, which tend to retrieve "EP" variables from Census unless they're unavailable (e.g., "EP_AGE17"). That might not be a problem for aggregating though, (for ZCTA3) if we use the retrieved "EP" values at the ZCTA level and aggregate them by the first 3 ZCTA digits for SVI calculation. I do agree that it would expand the use of SVI to include more geographic levels, and we'll try to incorporate that option in our development version/next release. 
Thanks again for using the package, and best of luck with your research!

**Reply by heli-xu** [reply]
Hi, we recently added a new feature in the package to support SVI calculation at customized geographic levels (#22) that may be related to your suggestion. You can install the development version from github if you're interested in trying it out. I'm still working on vignettes for the package, but for now this [post](https://heli-xu.github.io/svi-calculation/posts/validate_find_svi_x/validate_find_svi_x.html) has some information about the workflow and validation, and this [post](https://heli-xu.github.io/svi-calculation/posts/commute_zone/commute_zone.html) uses US commuting zone as an example with spatial analysis.

Looking at the previous message I think I somehow had the wrong impression that the "EP_" variables would have a common "total" count denominator, and that's why I thought it'd be ok to aggregate the EP values. Apologies for the confusion!

Please let me know if you have any questions or come across any issues. We'd appreciate any feedback and suggestions!
